### PR TITLE
Add integration options skipReservedkeyWordsEscaping, skipTracksTable

### PR DESCRIPTION
**Pull request by Jayachand**
## Description of the change

> Description here

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

**Review by sivashanmukh — DISMISSED** [review]
LGTM

File: warehouse/v1/util.js [hunk 1]
```diff
@@ -17,7 +20,8 @@ function safeTableName(provider, name = "") {
     tableName = tableName.toLowerCase();
   }
   if (
-    reservedANSIKeywordsMap[provider.toUpperCase()][tableName.toUpperCase()]
+    reservedANSIKeywordsMap[provider.toUpperCase()][tableName.toUpperCase()] &&
```
Comment (psrikanth88) [inline warehouse/v1/util.js:23]: reverse the order of checks to avoid extra check in case of blendo ?

File: warehouse/v1/util.js [hunk 2]
```diff
@@ -113,12 +122,41 @@ function transformName(provider, name = "") {
...
+    key = key.substr(0, 63);
+  }
+  return key.toLowerCase();
+}
+
+function toBlendoCase(name = "") {
+  return name.trim().toLowerCase();
+}
+
+function transformTableName(name = "", useBlendoCasing = false) {
```
Comment (psrikanth88) [inline warehouse/v1/util.js:150]: pass options here also and compute useBlendoCasing here? or set useBlendoCasing in the options at start of process itself

File: warehouse/v0/util.js [hunk 3]
```diff
@@ -29,7 +29,23 @@ function toSafeDBString(provider, name = "") {
   }
 }
 
-function safeTableName(provider, name = "") {
+function transformNameToBlendoCase(provider, name = "") {
```
Comment (psrikanth88) [inline warehouse/v0/util.js:32]: lets not make changes in v0 files. I don't think anybody is using this anymore

**Review by psrikanth88 — DISMISSED** [review]

File: warehouse/v1/util.js (hunk 1 above)
Comment (Jayachand) [reply warehouse/v1/util.js:23]: done

File: warehouse/v0/util.js (hunk 3 above)
Comment (Jayachand) [reply warehouse/v0/util.js]: removed v0/util

**Review by psrikanth88 — APPROVED** [review]

**Review by sivashanmukh — DISMISSED** [review]
LGTM

**Review by saikumarrs — APPROVED** [review]

File: CODEOWNERS [hunk 4]
```diff
@@ -1,4 +1,4 @@
-* @sanpj2292 @saikumarrs @arnab-p
```
Comment (saikumarrs) [inline CODEOWNERS:1]: CC @arnab-p

**Review by achettyiitr — APPROVED** [review]
LGTM

**Review by sivashanmukh — APPROVED** [review]
LGTM

**Pull request merged**